docs/Manual/04 Fusing.md

```diff
@@ -38,11 +38,11 @@ def _():
 
 schedule_simple = nest_simple.create_schedule()
 ```
-Note that each iteration in `schedule_simple` operates on all three arrays at once. Imagine that there is some computational advantage to operating on only two arrays at a time. For example, say that operating on all three arrays simultaneously creates excessive pressure on the computer's memory caches, which could hurt performance.
+Note that each iteration in `schedule_simple` executes simultaneously on all three arrays. However, there can be a case where concurrent operation on these arrays creates unbridled pressure on the computer’s cache, resulting in lower performance. In such a case, simultaneous operation on two arrays instead of three has a computational advantage.
 
-Therefore, we may want to first compute `C += A` and only then compute `C *= B`. Better yet, we may want to compute `C` in 4&times;4 blocks: first computing `C[0:4, 0:4] += A[0:4, 0:4]`; next computing `C[0:4, 0:4] *= B[0:4, 0:4]`; then moving on to the next block and computing `C[4:8, 0:4] += A[4:8, 0:4]`, and so on. Fusing gives us the flexibility to explore all of these possibilities, and more.
+Therefore, we first compute `C += A` and then compute `C *= B`.  Better yet, we may want to compute `C` in 4&times;4 blocks, i.e., first computing `C[0:4, 0:4] += A[0:4, 0:4]`; followed by computing `C[0:4, 0:4] *= B[0:4, 0:4]`; then moving on to the next block and computing `C[4:8, 0:4] += A[4:8, 0:4]`, and so on. In this respect, Fusing offers remarkable flexibility to explore all of these different execution possibilities. 
```

Comment:
On re-reading, this sentence may be broken into smaller sentences, for example:

"Therefore, we may want to first compute ....  Better yet, we may want to compute .... In the latter approach, we first compute .... Subsequently, we compute ....  Finally, we move on to the next block and compute ..., and so on.  This way, fusing offers remarkable flexibility..."